docs/examples/ns1/k8gb-cluster-ns1-eu-west-1.yaml

```diff
@@ -4,7 +4,11 @@ k8gb:
   edgeDNSServer: "169.254.169.253" # to handle splitbrain situation with TXT timestamp
   clusterGeoTag: "eu-west-1" # used for places where we need to distinguish between differnet Gslb instances
   extGslbClustersGeoTags: "us-east-1" # comma-separated list of external gslb geo tags to pair with
-  exposeCoreDNS: true # Create Service type LoadBalancer to expose CoreDNS
+
+coredns:
+  service:
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-type: nlb
```

Comment:
Can we move the coredns service configuration to separate PR maybe?

Reply:
@ytsarev you mean introduce scenario based on embedded service?

Reply:
I mean configuration change option change